tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java

```diff
@@ -358,7 +358,7 @@ public void testBuildWithTarget() throws Exception {
         Files.deleteIfExists(projectPath.resolve(".ballerina"));
 
         // Test ballerina build
-        String[] clientArgsForBuild = {"-o", genExecPath.resolve("foo.bal").toString(), "main.bal"};
```

Comment:
Shall we add a test case for this as well, since both ways should work for `build`?

Reply:
Added two separate tests to check order non-enforcement at https://github.com/ballerina-platform/ballerina-lang/pull/10173/files#diff-3285f46f75158ce485974d55778d7c90R81, thanks!